pkgs/applications/misc/hugo/deps.nix

```diff
@@ -72,16 +72,7 @@
     };
   }
   {
-    goPackagePath = "github.com/dchest/cssmin";
-    fetch = {
-      type = "git";
-      url = "https://github.com/dchest/cssmin";
-      rev = "fb8d9b44afdc258bfff6052d3667521babcb2239";
-      sha256 = "09sdijfx5d05z4cd5k6lhl7k3kbpdf2amzlngv15h5v0fff9qw4s";
-    };
-  }
-  {
-    goPackagePath = "github.com/dlclark/regexp2";
+    goPackagePath = "github.com/disintegration/imaging";
```

Comment:
This line seems suspect, maybe @monotux didn't update the deps with go2nix?

Reply:
I tried to merge the updates from master to this branch, but I guess I didn't do a very good job :)

Next time I attempt this I will try to _really_ understand how the tooling works.